packages/KTable/KTable.vue

```diff
@@ -108,6 +108,11 @@ table.k-table {
           text-decoration: underline;
         }
       }
+      button,
+      .k-button {
```

Comment:
we should be able to just declare these vars on the table itself, then you don't have to worry about k-button class ever changing

Reply:
I did do that at first but seemed cleaner to apply to the the actual button. Then when inspecting the code you see the override where its taking place. IDC though I can move if you think that makes more sense

Reply:
we can leave the specificity for now sure thing. we've changed k-button class once before, but doubt we'd do it again.